### DISCUSSION TITLE
Can't Control SplitView.Pane ColorPalette

### DISCUSSION THREAD

**Discussion by ghost** [AvaloniaUI/Avalonia#15684] [Q&A]
I'm new to Avalonia and have been away from C# for some years. I love that with Fluent Theme you can control Palette Accents & RegionColors, however I can't seem to find an attribute for the Pane side of the SplitView Control, they seem to change with RegionColour but not to the specified values, instead just the Content side changes. 

I couldn't find anything for the Pane side of the SplitView, Is it possible to alter this the same way we do with ColorPaletteResources?

    <FluentTheme.Palettes>
       <ColorPaletteResources x:Key="Light" Accent="LightBlue" RegionColor="White" ErrorText="Red" />
       <ColorPaletteResources x:Key="Dark" Accent="DarkBlue" RegionColor="DarkGray" ErrorText="Yellow"/>
    </FluentTheme.Palettes>

Answer: ChromeLow

**Comment by timunie** [comment]
I don't quite get what you are trying to do here? You can use `DynmaicResources` or wrap Pane content in an ContentControl or any other contianer.

**Reply by ghost** [reply]
I'm just trying to control the SplitView.Pane colour independently of SplitView.Content colour, using FluentTheme Pallettes. It seems like RegionColor doesn't cover the Pane side of SplitView, only the Content side. I tried setting BaseHigh, BaseLow etc but they also do not cover the SplitView.Pane side.

**Reply by timunie** [reply]
https://github.com/AvaloniaUI/Avalonia/blob/cfcc964cb2b6ee2713c81e5f90023556023574c9/src/Avalonia.Themes.Fluent/Controls/SplitView.xaml#L28

**Reply by ghost** [reply]
Yeah, ChromeLow was what I was looking for. Thanks
